app/http.php

```diff
@@ -728,29 +728,21 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
             }
 
             // Extract response
+            $executorResponse = json_decode($executorResponse ?? '{}', false);
+
             $execution = [];
-            $stdout = '';
-            $stderr = '';
             $res = '';
+            $stderr = $executorResponse->stderr ?? '';
+            $stdout = $executorResponse->stdout ?? '';
 
-            $executorResponse = json_decode($executorResponse ?? '{}', true);
-
-            switch (true) {
-                case $statusCode >= 500:
-                    $stderr = $executorResponse['stderr'] ?? '';
-                    $stdout = $executorResponse['stdout'] ?? '';
-                    break;
-                case $statusCode >= 100:
-                    $stdout = $executorResponse['stdout'] ?? '';
-                    $res = $executorResponse['response'] ?? '';
-                    if (is_array($res)) {
-                        $res = json_encode($res, JSON_UNESCAPED_UNICODE);
-                    }
-                    break;
-                default:
-                    $stderr = $executorResponse['stderr'] ?? '';
-                    $stdout = $executorResponse['stdout'] ?? '';
-                    break;
+            if ($statusCode >= 200 && $statusCode < 300) {
```

Comment:
```suggestion
            if ($statusCode >= 200 && $statusCode < 500) {
```
Why 300? Anything < 500 is a valid response right ?